packages/calypso-e2e/src/lib/pages/people-page.ts

```diff
@@ -0,0 +1,61 @@
+import { Page } from 'playwright';
+
+export type PeoplePageTabs = 'Team' | 'Followers' | 'Email Followers' | 'Invites';
+
+const selectors = {
+	// Navigation tabs
+	navTabs: '.section-nav-tabs',
+
+	// Invites
+	pendingInvite: ( email: string ) => `[title="${ email }"]`,
+	revokeInviteButton: 'button:text("Revoke invite")',
+	inviteRevokedMessage: 'span:text("Invite deleted.")',
+};
+
+/**
+ * Represents the Users > All Users page.
+ */
+export class PeoplePage {
+	private page: Page;
+
+	/**
+	 * Constructs an instance of the component.
+	 *
+	 * @param {Page} page The underlying page.
+	 */
+	constructor( page: Page ) {
+		this.page = page;
+	}
+
+	/**
+	 * Clicks on the navigation tab (desktop) or dropdown (mobile).
+	 *
+	 * @param {string} name Name of the tab to click.
+	 * @returns {Promise<void>} No return value.
+	 */
+	async clickTab( name: PeoplePageTabs ): Promise< void > {
+		// These navtabs do not change into a dropdown, unlike the navtabs present
+		// elsewhere in Calypso (eg. MediaPage).
+		await this.page.click( `${ selectors.navTabs } span:text("${ name }")` );
```

Comment:
Possibly better to use `:text-is()` here as the `:text()` matches also substrings.

Reply:
My understanding is that `:has-text()` is the one that matches substrings, and `:text()` behaves somewhere in between from `has-text()` and `text-is()`. With that said, I specify a CSS selector to narrow the selection range (and have used this technique throughout the library) so I do not think it is possible to accidentally match other elements on the page.